package/kernel/linux/modules/other.mk

```diff
@@ -83,6 +83,28 @@ endef
 $(eval $(call KernelPackage,ath3k))
 
 
+define KernelPackage/avm-wasp
+  SUBMENU:=$(OTHER_MENU)
+  TITLE:=AVM WASP remote Processor Support
+  KCONFIG:= \
+	CONFIG_REMOTEPROC=y \
```

Comment:
`CONFIG_REMOTEPROC` will be compiled into the kernel when this `KernelPackage` is activated.
I would prefer if you make this package depend on lantiq xrx200 target.
Please add this: `DEPENDS:=@TARGET_lantiq_xrx200`